### PR TITLE
Feature - Connect Looney

### PR DESCRIPTION
**Pull request by v-stamenova**
The following PR establishes the connection with the arduino.
No data is being currently sent from the app to the arduino - we need to decide as soon as possible when and what faces will be sent. The faces are hardcoded because it will be too time consuming and overengineering to make a database query. 

How to send data:
1. Make sure the arduino is connected - it gets connected via the main screen - We should probably add session flashing if it connected successfully. By make sure I mean put it within a try catch block the statement you're writing.
2. Go to the component/.svelte of your choice - not .server.js (those are serverside executions - not impossible but not tested by me)
3. Import the function for sending data
```js
import { sendDataViaBluetooth } from "$lib/bluetoothConnection.js";
```
4. Call the function
```js
await sendDataViaBluetooth([the face you want to send -> refer to gateway]);
```

A little sad that this cannot be tested without the arduino so I'll try to do most of it myself

### CODE REVIEW

**Comment by v-stamenova** [comment]
PS: It's 11 thousand lines of code because of the package-lock.json

File: arduino-sketch/bleAndMatrix.ino [hunk 1]
```diff
@@ -0,0 +1,86 @@
+// Configuration for the Arduino R4 WiFi
+
+#include <ArduinoBLE.h>
+#include "Arduino_LED_Matrix.h"
+
+BLEService customService("ee4e5719-e214-4217-ba3e-c5de249a3b3c");                                            // Custom service UUID
```
Comment (DanielBartha) [inline arduino-sketch/bleAndMatrix.ino:6]: I assume this is the hashed data connection for bluetooth?

**Review by DanielBartha — APPROVED** [review]
Awesome job! I wish I was smart enough to do stuff like this

File: arduino-sketch/bleAndMatrix.ino (hunk 1 above)
Comment (v-stamenova) [reply arduino-sketch/bleAndMatrix.ino:6]: That is something called [uuid](https://microchipdeveloper.com/xwiki/bin/view/applications/ble/introduction/bluetooth-architecture/bluetooth-host-layer/bluetooth-generic-attribute-profile-gatt/UUIDs/). I suggest you don't dive too much in it, I spent quite some time

**Pull request merged**